src/MusicStore.Spa/Infrastructure/SimpleMapper.cs

```diff
@@ -14,8 +14,12 @@ public class SimpleMapper
 
         public static TDest Map<TSource, TDest>(TSource source, TDest dest)
         {
+#if ASPNET50
+            return AutoMapper.Mapper.Map(source, dest);
+#else
             var map = (Func<TSource, TDest, TDest>)_mapCache.GetOrAdd(Tuple.Create(typeof(TSource), typeof(TDest)), _ => MakeMapMethod<TSource, TDest>());
             return map(source, dest);
+#endif
```

Comment:
Much better, even better would be to wrap in a different interface 

-----Original Message-----
From: "Kirthi Krishnamraju" <notifications@github.com>
Sent: ‎1/‎22/‎2015 9:14 PM
To: "aspnet/MusicStore" <MusicStore@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [MusicStore] Removed BuddyType attribute and addedModelMetadatType  (#369)

In src/MusicStore.Spa/Infrastructure/SimpleMapper.cs:
>              var map = (Func<TSource, TDest, TDest>)_mapCache.GetOrAdd(Tuple.Create(typeof(TSource), typeof(TDest)), _ => MakeMapMethod<TSource, TDest>());
>              return map(source, dest);
> +#endif
@yishaigalatzer - Moved AutoMapper mapping to SimpleMapper Map method. Does this look better?
—
Reply to this email directly or view it on GitHub.=